### DISCUSSION TITLE
multiple whitelist_from urls in cross-account filter

### DISCUSSION THREAD

**Discussion by mattheidelbaugh** [cloud-custodian/cloud-custodian#7417] [General]
Is there a workaround that would allow me to use multiple urls in a cross-account filter or otherwise allow to whitelist_from multiple files? (specifically for ecr in this case but most likely would be used for other resources in the future as well).  I have two json files with the same format that I want to whitelist from, but it seems like the current filter configuration only allows for one.  

Something like this is what I would like:
                - type: cross-account
                  whitelist_from:
                    expr: accounts.*.account
                    url: 
                    - s3://mybucket/my-accounts.json
                    - s3://mybucket/my-old-accounts.json

**Comment by ajkerrigan** [comment]
Hi @mattheidelbaugh - you're correct that `url` is a single string, and I can't see a useful way to chain multiple `cross-account` filters either. As for a workaround, if it's feasible I'd suggest merging the two files into something like `s3://mybucket/my-combined-accounts.json` and using that in your policy.
